assets/javascripts/discourse/components/post-menu/toggle-translation-button.gjs

```diff
@@ -12,6 +12,7 @@ export default class ToggleTranslationButton extends Component {
 
   @service modal;
   @service translator;
+  @service siteSettings;
```

Comment:
Did you mean to use this in `showButton` maybe? It's not used elsewhere in this file